### PR TITLE
net-news-wire: init at 6.1.4

### PR DESCRIPTION
**Pull request by jakuzure**
## Description of changes

- NetNewsWire: init at 6.1.4
https://github.com/Ranchero-Software/NetNewsWire/tree/main
- add jakuzure to maintainers

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/feedreaders/NetNewsWire/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  src = fetchzip {
+    url =
+      "https://github.com/Ranchero-Software/${pname}/releases/download/mac-${version}/${pname}${version}.zip";
+    hash = "sha256-9RuvMklwFBi7KbR0P2HU1Mpe5t78W7CwtoxAkU5E2og=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/Applications
+    cp -R NetNewsWire.app $out/Applications/
+  '';
```
Comment (h7x4) [inline pkgs/applications/networking/feedreaders/NetNewsWire/default.nix:16]: [Remember the hooks (see checklist)](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#new-packages)

```suggestion
  installPhase = ''
    runHook preInstall
    
    mkdir -p $out/Applications
    cp -R NetNewsWire.app $out/Applications/
    
    runHook postInstall
  '';
```

File: pkgs/applications/networking/feedreaders/NetNewsWire/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  meta = with lib; {
+    description = "RSS reader for macOS and iOS.";
```
Comment (h7x4) [inline pkgs/applications/networking/feedreaders/NetNewsWire/default.nix:19]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

```suggestion
    description = "RSS reader for macOS and iOS";
```

**Review by h7x4 — CHANGES_REQUESTED** [review]
Just some small comments

**Comment by jakuzure** [comment]
@h7x4 Thank you very much for your review and all the links, I adapted your suggested changes

File: pkgs/applications/networking/feedreaders/net-news-wire/default.nix [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    mkdir -p $out/Applications
+    cp -R NetNewsWire.app $out/Applications/
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "RSS reader for macOS and iOS";
+    longDescription = ''
+      It’s like podcasts — but for reading.
+      NetNewsWire shows you articles from your favorite blogs and news sites and keeps track of what you’ve read.
```
Comment (h7x4) [inline pkgs/applications/networking/feedreaders/net-news-wire/default.nix:27]: This is probably not a big deal, but just for the sake of being safe, maybe we should swap out these ticks with more ASCII-compliant ones?

```suggestion
      It's like podcasts — but for reading.
      NetNewsWire shows you articles from your favorite blogs and news sites and keeps track of what you've read.
```

**Review by h7x4 — APPROVED** [review]

**Review by FliegendeWurst — APPROVED** [review]

**Comment by jakuzure** [comment]
@FliegendeWurst @h7x4 Thanks for the approvals, could anybody merge this PR?

**Review by wegank — APPROVED** [review]

**Pull request merged**